Resources/Prototypes/DeltaV/Entities/Objects/Misc/rubber_stamp.yml

```
@@ -11,6 +11,20 @@
   - type: Sprite
     sprite: DeltaV/Objects/Misc/stamps.rsi
     state: stamp-psychologist
+
+- type: entity
+  name: pharmacist's rubber stamp
+  parent: RubberStampBase
+  id: RubberStampPharmacist
+  suffix: DO NOT MAP
```

```suggestion
  suffix: DO NOT MAP
  noSpawn: true
```
DO NOT MAP should be set to noSpawn to avoid it being mapped